src/test/java/bobby/parser/TestParser.java

```diff
@@ -0,0 +1,88 @@
+package bobby.parser;
+
+import bobby.command.Command;
+import bobby.exceptions.*;
+import bobby.tasks.Task;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
```

Comment:
I think static imports should be ordered first, followed by org.*, then the remaining imports.
Ordering of imports is shown [here](https://se-education.org/guides/tutorials/intellijCodeStyle.html).